### PR TITLE
add_read_file

### PR DESCRIPTION
**Pull request by koval1off**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,12 +2,21 @@
 from datetime import datetime
 
 
+with open("darabase.txt", "w") as f:    # just to create the file automatically
```
Comment (koval1off) [inline main.py]: don't know if it's really needed, but it's better that creating this file yourself

File: main.py [hunk 2]
```diff
@@ -2,13 +2,18 @@
...
 
 
+with open("darabase.txt", "w") as f:    # just to create the file automatically 
+    f.write("2222:3333")
+
+
 def check_id_and_pin(user_id, user_pin):
-    if user_id == 2222 and user_pin == 3333:
-        return True
+    with open("database.txt", "r") as base:
```
Comment (koval1off) [inline main.py:11]: in the commits you can check how I did it before. 
But after I tried to think about the task a little bit more, and here is the result

File: main.py [hunk 3]
```diff
@@ -2,11 +2,18 @@
...
 
-    return False
+
+def check_id_and_pin(user_id, user_pin):
+    with open("database.txt", "r") as base:
+        for line in base.readlines():
+            line = line.split(":")
+            if str(user_id) in line[0]:
+                if str(user_pin) in line[1]:
+                    return True
```
Comment (koval1off) [inline main.py:16]: It works good now for multiple IDs and PASSes that can be stored on the database.

File: main.py (hunk 1 above)
Comment (Alex-Krykun) [reply main.py]: database

**Review by Alex-Krykun — APPROVED** [review]
nice job!

**Pull request merged**